### DISCUSSION TITLE
Execute  'curl -fsSL https://code-server.dev/install.sh | sh -s -- --dry-run ' command and report an error !

### DISCUSSION THREAD

**Discussion by CCJude** [coder/code-server#6825] [Q&A]
curl: (35) error:0A000126:SSL routines::unexpected eof while reading

**Comment by CCJude** [comment]
![无标题](https://github.com/coder/code-server/assets/94972533/c5ddd96e-9ebd-43ae-b1e3-697d4d4101c4)

**Reply by code-asher** [reply]
Is your cURL a recent version?  Or maybe the OpenSSL version?  I tried just now and it is working for me.  Does it load in your browser?

**Comment by CCJude** [comment]
ccjude@ubuntu-develop:~$ curl --version
curl 7.81.0 (x86_64-pc-linux-gnu) libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.17
Release-Date: 2022-01-05
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets zstd

**Reply by code-asher** [reply]
Hmm maybe the version is unrelated after all, I just tried an older version in CentOS 8:

```
# curl  --version
curl 7.61.1 (x86_64-redhat-linux-gnu) libcurl/7.61.1 OpenSSL/1.1.1g zlib/1.2.11 nghttp2/1.33.0
Release-Date: 2018-09-05
Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtsp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy Metalink
```

But it worked for me:

```
# curl -fsSL https://code-server.dev/install.sh | sh -s -- --dry-run
CentOS Linux 8
Installing v4.89.1 of the amd64 rpm package from GitHub.
```

Sorry, wish I could be more help.
